### PR TITLE
Elaborated a contribution rules. Resolved #8

### LINKED ISSUE #8: Elaborate a contribution rules.
In README.md file.

### PR DESCRIPTION
**Pull request by mariuszkumor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+# How to contribute to the repository
+Contributors wanted to join Power Flows project have to comply with a few rules: 
+
+* A preferred IDE is IntelliJ IDEA with default formatting styles
+* Committed source code must be high quality
+* Committed source code must be formatted
+* A commit has to solve an issue from the product backlog (if missing the issue there, please add previously)
+* In order to start contribution, a contributor has to fork the project
+* Once an issue is fixed, a contributor has to create a commit to his forked repository
+* In order to merge changes to the project repository, contributor has to create a pull request with the only one commit.
```
Comment (stlhrt) [inline README.md]: 'with single commit'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
-# powerflows-dmn-model
+# About Power Flows DMN Model
 Power Flows DMN Model
+
+# How to contribute to the repository
+Contributors wanted to join Power Flows project have to comply with a few rules:
```
Comment (stlhrt) [inline README.md]: 'wanting to' or 'whishing to'

File: README.md [hunk 3]
```diff
@@ -1,2 +1,18 @@
...
+# About Power Flows DMN Model
 Power Flows DMN Model
+
+# How to contribute to the repository
+Contributors wanted to join Power Flows project have to comply with a few rules: 
+
+* A preferred IDE is IntelliJ IDEA with default formatting styles
+* Committed source code must be high quality
+* Committed source code must be formatted
+* A commit has to solve an issue from the product backlog (if missing the issue there, please add previously)
```
Comment (stlhrt) [inline README.md]: ' if the issue is missing, add one before creating pr. Refer the issue in commit summary.'

File: README.md [hunk 4]
```diff
@@ -1,2 +1,18 @@
...
+* In order to start contribution, a contributor has to fork the project
```
Comment (stlhrt) [inline README.md]: This one is kinda obvious for all github users

Comment (mariuszkumor) [reply README.md]: Yes and no, since I got many times questions, how to create a branch in a project repo?

File: README.md (hunk 1 above)
Comment (mariuszkumor) [reply README.md]: Improved

File: README.md (hunk 2 above)
Comment (mariuszkumor) [reply README.md]: Improved

File: README.md (hunk 3 above)
Comment (mariuszkumor) [reply README.md]: Improved

File: README.md [hunk 5]
```diff
@@ -1,2 +1,18 @@
-# powerflows-dmn-model
+# About Power Flows DMN Model
 Power Flows DMN Model
+
+# How to contribute to the repository
+Contributors wishing to join Power Flows project have to comply with a few rules: 
+
+* A preferred IDE is IntelliJ IDEA with default formatting styles
+* Committed source code must be high quality
```
Comment (mateuszbochniak) [inline README.md]: Może dodać o pokryciu UT?

Comment (mariuszkumor) [reply README.md]: Good idea, done.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,19 @@
...
+
+* A preferred IDE is IntelliJ IDEA with default formatting styles;
+* Committed source code must be high quality;
+* Committed source code must be formatted;
+* A commit has to solve an issue from the product backlog. If the issue is missing, please add one before creating the pull request. Refer the issue in commit summary;
+* Unit tests coverage should be at 80% or higher;
+* In order to start contribution, a contributor has to fork the project;
+* Once an issue is fixed, a contributor has to create a commit to his forked repository;
+* In order to merge changes to the project repository, contributor has to create a pull request with single commit.  
+
```
Comment (stlhrt) [inline README.md]: no source encoding and EOL requirements ?

Comment (mariuszkumor) [reply README.md]: Good idea, added.

**Pull request merged**